code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -313,6 +313,8 @@
 		return JOB_UNAVAILABLE_ACCOUNTAGE
 	if(job.required_playtime_remaining(client))
 		return JOB_UNAVAILABLE_PLAYTIME
+	if(latejoin && !job.special_check_latejoin(client))

```

Comment:
Why are you passing the client in here. Either define it at all levels or not.

Reply:
now defined on AI too.